serverless google cloud run (#15359)
DataDog/documentation: content/en/serverless/google_cloud_run/_index.md

---
title: Google Cloud Run
kind: documentation
---

## Overview
Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. 

## Build your container

If you are using a Dockerfile to build your application, complete the following:

1. Copy the [Datadog `serverless-init` binary][1] into your Docker image.

2. Use the ENTRYPOINT instruction to run the `serverless-init` binary as your Docker container is initiated.

3. Use the CMD instruction to run your existing application and other required commands as arguments.

> The following are simple examples for how to complete these three steps. You may need to adjust these examples depending on your existing Dockerfile setup.

Reviewer: ```suggestion
The following are examples of how to complete these three steps. You may need to adjust these examples depending on your existing Dockerfile setup.
```
